macOS/PixelDefinitions/pixels/navigation.json5

```diff
@@ -83,10 +92,16 @@
             }
         ]
     },
-    "siteLoadingPerformance": {
+    "m_mac_site_loading_performance": {
         "description": "Fired when CSS pushes privacy-aware performance metrics via expandedPerformanceMetricsResult notification",
         "owners": ["diegoreymendez"],
         "triggers": ["page_load"],
+        "suffixes": [
+            {
+                "description": "Sampling percentage (between 1 and 100 inclusive)",
+                "pattern": "sample(?:100|0?[1-9]|[1-9]\d)$"
```

Comment:
Updated the regex to hopefully match - I don't have a personal dev instance available to test it with, but I've needed to use the temp instances enough lately that I'm going to request a personal one.